packages/example/src/js/race/PushTheming.tsx

```diff
@@ -0,0 +1,249 @@
+import { NativeNavigation, ViewUpdate } from 'capacitor-native-navigation'
+import { useNativeNavigationViewContext } from 'capacitor-native-navigation-react'
+import React, { useCallback, useEffect, useLayoutEffect, useState } from 'react'
+import { Route, useNavigate } from 'react-router-dom'
+import TallContent from '../examples/TallContent';
+
+
+const SetBodyAttribute: React.FC<{ attribute: string; value: string | null }> = ({ attribute, value }) => {
```

Comment:
Let's add some documentation to this component to explain what this is an example of, and some docs through the code to explain.